students/HABTAMU/session02/series.py

```diff
@@ -0,0 +1,55 @@
+# Fibonacci Series Exercise.
+
+def fibonacci(n):
+    a, b = 0, 1
+
+    if n <= 0:
+        return 1
+    else:
+        for i in range(n):
+            print(a, end=" ")
+
+            c = a + b
+            a = b
+            b = c
```

Comment:
Interesting solution!  The assignment only required the return of the nth value, but this works.